Dockerfile

```diff
@@ -48,16 +52,20 @@ RUN curl -sSL https://install.python-poetry.org \
   python3 -q
 WORKDIR /src
 RUN python3 -m venv $VIRTUAL_ENV
-RUN poetry install
+RUN poetry install && rm -rf /tmp/cache
+
+FROM poetry AS code
 
 # Add project
 USER root
+# copy in trusted certs
+COPY --chmod=644 certs/*.crt /usr/local/share/ca-certificates/
+RUN update-ca-certificates
 COPY . /src
 WORKDIR /src
 RUN mkdir -p /src/staticfiles
 
-RUN apt-get clean && apt-get purge
-
+FROM code as final
```

Comment:
Not really important, but Docker throws a warning that `AS` is not capitalized here